Dockerfile

```diff
@@ -1,27 +1,10 @@
-#
-# ML-Workbench Dockerfile
-#
-
-FROM aleenaraj/raj_ubuntu
-MAINTAINER Aleena Raj "aleenar@ilimi.in"
-
-# Setting up DS_DATA_HOME
-RUN mkdir /home/DS_DATA_HOME
-RUN mkdir /home/ML-Workbench
-
-# Setting the working directory
-WORKDIR /home
-
-ADD . /home/ML-Workbench
-
-ADD google_cred.json /home
-ADD credentials.ini /home
-
-# Setting the environment variable
-ENV GOOGLE_APPLICATION_CREDENTIALS /home/google_cred.json
-
-RUN pwd
-
-#Running MLWB
-RUN pip3 install -r /home/ML-Workbench/requirements.txt
+FROM python:3.6.5 as buildingml
+WORKDIR /home/
+WORKDIR /home/ml-workbench/
+RUN apt update && git clone https://github.com/project-sunbird/sunbird-ml-workbench -b release-1.15 .
+RUN bash -x build.sh \
+        && pip install bin/daggit-0.5.0.tar.gz \
```

Comment:
please remove this line till 8.
And copy the artifacts from the container, to a final container, from which we'll run the MLWB.

Reply:
we will do it as a multistage container with build in one container and installer dependencies, ml-workbench in another container